hypothesis/report
# Add a thin shim of views to create the structure we want in the DB

For:

 * https://github.com/hypothesis/report/issues/108

We want to split up the views which are dumped in the `public` schema into `h` and `lms` schemas so each product has it's own.

This PR:

 * Adds the new schema
 * Splits up the data task files by product
 * Adds thin views which with the new names in the new schema, pointing back to the real views in `public`

This **does not** actually move the underlying views. This should allow us to move as gracefully as Metabase will allow, and then move the views underneith later on.

## Testing notes

 * Start all the things
 * Run the create from scratch (skip python to speed this up)
 * `make sql`
 * `select * from h.authorities` etc. - Basically have a nose about and see the views are there
